### PR TITLE
compas_view2 add recipe

### PR DESCRIPTION
**Pull request by brgcode**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by brgcode** [comment]
@conda-forge-admin package is ready for review

**Comment by Licini** [comment]
LGTM

**Comment by brgcode** [comment]
@conda-forge-admin please ping team

File: recipes/compas_view2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "compas_view2" %}
+{% set version = "0.2.1" %}
+{% set hash = "ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/compas_view2/meta.yaml:10]: You don't need this line.

File: recipes/compas_view2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "compas_view2" %}
+{% set version = "0.2.1" %}
+{% set hash = "ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324" %}
```
Comment (ocefpaf) [inline recipes/compas_view2/meta.yaml:3]: You don't need a jinja variable that will be used only once.

Comment (ocefpaf) [inline recipes/compas_view2/meta.yaml:3]: You don't need a jinja variable that will be used only once.

**Review by ocefpaf — APPROVED** [review]
Please try to use [grayskull](https://conda-forge.org/blog/posts/2020-03-05-grayskull/) to generate your recipes. That may help you get the latest recipe syntax easier.

**Review by ocefpaf — COMMENTED** [review]
Please try to use [grayskull](https://conda-forge.org/blog/posts/2020-03-05-grayskull/) to generate your recipes. That may help you get the latest recipe syntax easier.

**Review by ocefpaf — COMMENTED** [review]
Please try to use [grayskull](https://conda-forge.org/blog/posts/2020-03-05-grayskull/) to generate your recipes. That may help you get the latest recipe syntax easier.

**Comment by brgcode** [comment]
@ocefpaf thanks for the advice. will generate starting recipe with grayskull in the future. do you think this can be merged now as is, or is there something else that needs updating first?

File: recipes/compas_view2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324
+
+build:
+  number: 0
+  skip: true  # [linux or py<36 or py>38]
```
Comment (ocefpaf) [inline recipes/compas_view2/meta.yaml:14]: Why are you skipping Linux?

Comment (brgcode) [reply recipes/compas_view2/meta.yaml:14]: got an error during the builds saying that OpenGL could not be found. we had a similar problem with the latest Mac OS but there the solution was straightforward. not sure how to fix this on linux so i figured to skip for now...

Comment (brgcode) [reply recipes/compas_view2/meta.yaml:14]: actually the error only pops up during the test phase...

Comment (brgcode) [reply recipes/compas_view2/meta.yaml:14]: the solution to the same problem on Mac was this https://github.com/compas-dev/compas_view2/blob/main/src/compas_view2/__init__.py#L20

**Comment by brgcode** [comment]
@conda-forge-admin please rerender

File: recipes/compas_view2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324
+
+build:
+  number: 0
+  skip: true  # [py<36 or py>38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ cdt('mesa-libGL-devel') }}  # [linux]
```
[the remark this answers is not in the document]
Comment (brgcode) [reply recipes/compas_view2/meta.yaml:19]: thanks! seems to work now...

[the remark this answers is not in the document]
Comment (brgcode) [reply recipes/compas_view2/meta.yaml:19]: something i need to do now to get this merged?

**Comment by brgcode** [comment]
@conda-forge/staged-recipes package is ready

File: recipes/compas_view2/meta.yaml (hunk 4 above)
Comment (chrisburr) [inline recipes/compas_view2/meta.yaml:19]:
```suggestion
```

Comment (brgcode) [reply recipes/compas_view2/meta.yaml:19]: but if i remove this the linux build fails because it can't find OpenGL...

Comment (chrisburr) [reply recipes/compas_view2/meta.yaml:19]: I think you added the yum requirements after the CDT was added? The CDT should only be needed if you're linking against OpenGL.

**Pull request merged**